def validate_num(num):
    valid_nums = [1, 2, 3]
    if num in valid_nums:
        pass
    else:
        raise Exception(f'Enter valid number ( 1 ), ( 2 ), ( 3 )')

